### DISCUSSION TITLE
Issues with configuration?

### DISCUSSION THREAD

**Discussion by greg-michael** [PowerShell/Win32-OpenSSH#2058] [Q&A] [answered]
I'm running OpenSSH through the Windows Server 2019 add-on. The server works, and I can establish a connection from an outside Cisco switch to the server and login successfully. When attempting to run the following command I get a weird error at the end.
`copy running-config scp://<IP address of SSH Server>/agent-2.tmp`

The file appears to be copied, but the end of the process throws this weird error:
`***************#copy running-config scp://<user>@<IP_Address>/agent-2.tmp`
`Address or name of remote host [<IP_Address>]? `
`Destination username [<user>]? `
`Destination filename [agent-2.tmp]? `
`Writing agent-2.tmp `
`Password: `
`S ink: C0644  101043 agent-2.tm p`
`! scp.exe: debug1: fd 0 clearing O_NONBLOCK`
`%scp: agent-2.tmp: add_mark_of_web failed`

I am at a loss on how to even begin troubleshooting this as I don't manage network devices. I'm attempting to upgrade a vendor application that uses this SSH server to facilitate SCP/SFTP transfers from network devices back to the application. The vendor does not provide any specific application to use, only that an SCP/SFTP/SSH program be used to facilitate the communications.

I've opened a support case with the vendor, but have not been able to get any forward progress. I'm hopeful that anyone here will either have an idea, some thoughts on how to troubleshoot the process from both the SSH server as well as the Cisco device, or some other suggestion(s) that I haven't thought of.

Thanks in advance!
Greg

**Comment by tgauth** [comment, marked as answer]
I think the SSH client just needs to be updated. The latest GitHub release removes mark-of-the-web (MOTW) so this error should not occur anymore. It can be downloaded from https://github.com/PowerShell/Win32-OpenSSH/releases/tag/v9.2.2.0p1-Beta or via WinGet. 

Confirm the latest release is being used by running `ssh -V`

**Reply by greg-michael** [reply]
`C:\Windows\system32>ssh -V`
`OpenSSH_for_Windows_9.1p1, LibreSSL 3.6.1`

It looks like it might be worthwhile to upgrade the installation to see if that resolves the issue.

Thanks!

**Reply by greg-michael** [reply]
Upgrading to 9.2.2.0 corrected the issue as described! Thanks @tgauth!
